src/config/configuration.ts

```diff
@@ -1,5 +1,7 @@
 export default () => ({
-  port: parseInt(process.env.PORT, 10) || 3000,
+  port: parseInt(process.env.PORT, 10) || 3001,
+  rabbitURL: process.env.RABBITMQ_URL,
+  fullURL: process.env.FULL_URL || `localhost:${process.env.PORT || 3000}`,
```

Comment:
line 4 was removed completely. As we will not use the fullURL anymore. We will send the uuid of the file instead of the full url. This's after discussion with Ayoub